spec/models/billing_contract_spec.rb

```diff
@@ -1,21 +1,35 @@
 require 'rails_helper'
 
 RSpec.describe BillingContract, type: :model do
-  let!(:billing_contract) { FactoryBot.build(:billing_contract) }
+  let!(:billing_contract) { FactoryBot.create(:billing_contract) }
```

Comment:
do you need create instance before all specs?
if you need it in one test, you need to just call object before you you write `expec` blocks